python/api/src/araali/__main__.py

```diff
@@ -510,6 +511,52 @@ def fw_config(args):
     knobs = api.API().get_fw_config(zone=args.zone, tenant=args.tenant)
     print(yaml.dump(knobs))
 
+def quickstart(args):
+    if args.quickstart_subparser_name == "vm":
+        success, result = _aws.cf_launch_fortified_vm(stack_name="araali-quickstart-vm-stack", email=args.email, key_pair=args.key)
+        if not success:
+            print("Quickstart setup failed")
+            return
+        return result
+    elif args.quickstart_subparser_name == "eks":
+        EKS_STACK_NAME = "araali-quickstart-stack"
+        EKS_CLUSTER_NAME = "araali-quickstart-eks"
+        cluster_name = args.name
+        launch_cluster = args.launchcluster
+        if cluster_name == "":
+            launch_cluster = True
+            cluster_name = EKS_CLUSTER_NAME
+        # Launch cluster is requested
+        if launch_cluster:
+            success, result = _aws.cf_launch_eks_cluster(stack_name=EKS_STACK_NAME, cluster_name=cluster_name, availability_zones=args.availabilityzones)
+            if not success:
+                print("Error: Quickstart EKS cluster setup failed")
+                return
+            # Wait until cluster is set up
+            counter = 0
+            while(True):
+                print("Waiting for cluster create - %d/20" %counter)
+                if _aws.cf_validate_stack_creation(EKS_STACK_NAME):
+                    break
+                if counter >= 20:
+                    print("Error: Quickstart EKS cluster setup failed")
+                    return
+                counter += 1
+                time.sleep(60 * 3)
```

Comment:
too much sleeping. I would sleep 30s or even 10s. Lets say your cluster is ready and you just missed it. You are making me wait another 30 minutes, which is ages in computer time